functions/index.js

```diff
@@ -126,39 +126,60 @@ function handleMessage(sender_psid, received_message) {
 }
 
 function handlePostback(sender_psid, received_postback) {
+
   console.log('ok')
-   let response;
+  var userPublicProfile = "https://graph.facebook.com/v2.6/" + sender_psid + "?fields=first_name,last_name,profile_pic&access_token="+ PAGE_ACCESS_TOKEN;
+  let response;
   let payload = received_postback.payload;
 
-if (payload === 'GET_STARTED_PAYLOAD') {
-    response = {
-      "attachment": {
-        "type": "template",
-        "payload": {
-          "template_type": "generic",
-          "elements": [{
-            "title":"Hi {{user_first_name}}!",
-            "subtitle": "I'm Duber. I can help you find people with answers to your questions.",
-            "buttons": [
-              {
-                "type": "postback",
-                "title": "Complete my Profile",
-                "payload": "PAYLOAD_COMPLETE_MY_PROFILE",
+  if (payload === 'GET_STARTED_PAYLOAD') {
+      let user_first_name 
+
+      request({
+        'url': userPublicProfile,
+        'json':true
+      }, function(error, res, body){
+        if(error){
```

Comment:
Add space before the `(` and after the `)` so that its if `if (error) {` check this style guide for reference: https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace Do the same for the other occurances